man/journald.conf.xml

```diff
@@ -358,6 +358,28 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>SyslogFacility=</varname></term>
+        <listitem><para>Sets the syslog facility to use when logging
+        to syslog. One of <option>kern</option>,
```

Comment:
I might have misunderstood #5937 as well. I think it would be great if @rfrail3 could elaborate what #5937 is about.

Anyway, currently `MaxSyslogFacility` doesn't seem to affect what `journald` sends to `syslog/kmsg/console/wall`. Also, it's not possible to pass `systemd.journald.max_syslog_facility`. All other settings might be set in that way.